Document/0x04a-Mobile-App-Taxonomy.md

```diff
@@ -50,6 +50,6 @@ PWAs are supported by Android and iOS, but not all hardware features are yet ava
 
 ## What's Covered in the Mobile Testing Guide
 
-Throughout this guide, we will focus on apps for the two platforms dominating the market: Android and iOS. Mobile devices are currently the most common device class running these platforms – increasingly however, the same platforms (in particular, Android) run on other devices, such as smartwatches, TVs, car navigation/audio systems, and other embedded systems.
+Throughout this guide, we will focus on apps for the two platforms dominating the market: Android and iOS. Mobile devices are currently the most common device class running these platforms. However, the same platforms (or adapted versions of them) run on other devices, such as smartwatches, tablets, TVs, car navigation/audio systems, and other embedded systems.
```

Comment:
The initial intention was just to remove the dash but thanks for this remark. I agree and it made me question the whole sentence. I've rewritten it to reflect the focus of the MSTG a bit better: all in the first sentence now, the rest is additional information.

```suggestion
Throughout this guide, we will focus on apps for Android and iOS running on smartphones. These platforms are currently dominating the market and also run on other device classes including tablets, smartwatches, smart TVs, automotive infotainment units, and other embedded systems. Even if these additional device classes are out of scope, you can still apply most of the knowledge and testing techniques described in this guide with some deviance depending on the target device.
```